plan9/srv9p/example/netshell/main.go

```diff
@@ -236,8 +238,8 @@ func newConsole(r io.Reader, w io.Writer) *console {
 // consctlClose handles a close of /dev/consctl; it turns raw mode off.
 func (c *console) consctlClose() {
 	c.mu.Lock()
+	defer c.mu.Unlock()
```

Comment:
this is just for consistency with the style below. I don't care strongly, but I've been forcing myself to do this more often ever since it compiles to the same thing. Not like this is hot anyway.